kqueen/utils/loggers.py

```diff
@@ -4,12 +4,22 @@
 import logging
 
 
-def setup_logging(path='kqueen/utils/logger_config.yml', default_level=logging.INFO):
+def setup_logging(path, debug_mode):
+    default_level = 'INFO'
 
     if os.path.exists(path):
         with open(path, 'rt') as f:
             try:
                 config = yaml.safe_load(f.read())
+                loggers = config['loggers']
+                for logger, value in loggers.items():
+                    if debug_mode:
+                        loggers[logger]['level'] = 'DEBUG'
+                    else:
+                        current_level = value.get('level')
+                        is_level_not_set = value['level'] == 0 or current_level is None
```

Comment:
Can you please refactor this two lines to be more readable. I understand this code but it's hard to read it on the first look.

However, this isn't a blocker.

Reply:
Next time advice me how I can refactor a piece of code that you don't like